setup.py

```diff
@@ -75,7 +75,7 @@ def long_description():
         'Programming Language :: Python :: Implementation :: CPython',
         'Programming Language :: Python :: Implementation :: PyPy',
     ],
-    python_requires='>=3.5, <=3.8',
+    python_requires='>=3.5, <3.9',
```

Comment:
If you look, this was the form we had in a previous iteration. I accepted feedback that it should be equivalent, except it doesn't take minor versions into account :-1: